sql-prepare-plan-cache.md

```diff
@@ -5,7 +5,7 @@ summary: Learn about SQL Prepare Execution Plan Cache in TiDB.
 
 # SQL Prepare Execution Plan Cache
 
-In the current development release of TiDB, the execution plan of prepared statements is cached by default. This includes both forms of prepared statements:
+TiDB supports execution plan caching for `Prepare` / `Execute` queries. This includes both forms of prepared statements:
```

Comment:
```suggestion
TiDB supports execution plan caching for `Prepare` and `Execute` queries. This includes both forms of prepared statements:
```

Reply:
The plan cache is not for "Prepare and Execute", it is for "Prepare" / "Execute" combination.

Reply:
Does "/" means AND or OR here?

Reply:
Users would first use `Prepare` for a query, then use several consecutive `Execute` for the prepared statement. 

The feature is to cache the plan, while `Prepare` does not generate a plan actually, the plan is built in the first `Execute` and cached then, so strictly we cannot say "TiDB supports execution plan caching for `Prepare` queries".

Reply:
Shall we remove `Prepare` here? "TiDB supports execution plan caching for `Execute` queries."?

Reply:
The feature was called "prepare-plan-cache" historically...

Reply:
Changed to "`Prepare` and `Execute`" now.